Eluant/LuaRuntime.cs

```diff
@@ -492,9 +493,9 @@ private void ProcessReleasedReferences()
             }
         }
 
-        private void LoadString(string str)
+        private void LoadBuffer(string str, string name)
```

Comment:
Just curious if there's a reason these methods were renamed, since this breaks both ABI and source compatibility.